### PR TITLE
chore(core) - move content metadata to superdesk

### PR DESCRIPTION
**Pull request by amagdas**

### CODE REVIEW

File: server/apps/archive/common.py [hunk 1]
```diff
@@ -22,13 +22,12 @@
 from superdesk.utc import utcnow, get_expiry_date
 from settings import SERVER_DOMAIN, OrganizationNameAbbreviation
 from superdesk import get_resource_service
-from apps.content import metadata_schema
+from superdesk.metadata.item import metadata_schema
 from superdesk.workflow import set_default_state, is_workflow_state_transition_valid
 import superdesk
 from apps.archive.archive import SOURCE as ARCHIVE
-from apps.content import PACKAGE_TYPE, TAKES_PACKAGE
+from superdesk.metadata.item import PACKAGE_TYPE, TAKES_PACKAGE, not_analyzed
```
Comment (petrjasek) [inline server/apps/archive/common.py]: not sure takes should be there, any other options?

Comment (amagdas) [inline server/apps/archive/common.py]: Yup, will move some of the keys to apps and move some stuff from `apps.archive.common.py`

Comment (sivakuna-aap) [inline server/apps/archive/common.py]: Takes isn't specific to AAP. I think it can be part of chore

Comment (petrjasek) [inline server/apps/archive/common.py]: yep, but it could be in 'superdesk.metadata.takes'

Comment (sivakuna-aap) [inline server/apps/archive/common.py]: :+1:

Comment (mdhaman) [inline server/apps/archive/common.py]: :+1:

**Comment by ioanpocol** [comment]
:+1:

**Comment by petrjasek** [comment]
:+1:

**Pull request merged**